changelog.d/20250512_143605_sirosen_remove_default_scopes.rst

```diff
@@ -0,0 +1,8 @@
+Removed
```

Comment:
Based on my past experiences, I recommend that we create a `Breaking changes` section and capture changes like these in that section (even if via duplication of the bullet item's text).